simulator/simulator.py

```diff
@@ -1,22 +1,50 @@
+from io import BytesIO
+import os
 import pygame
 from pygame.locals import QUIT
 import time
+from dotenv import load_dotenv
+import aiohttp
+
 from peripheral.constants import (
-    BLUE,
+    AUDIO,
     CAPTION,
-    GREEN,
+    IMAGE,
     INITIALIZATION_ERROR,
     SCREEN_FILL,
     READING_ERROR,
     WHITE,
     RED,
     POINTER_REFRESH_TIME,
-    GENERIC_ERROR
+    GENERIC_ERROR,
+    WIDTH,
+    HEIGHT,
+    SCENARY
 )
 
+load_dotenv()
+
 red_points = []
 
-def simulator(data):
+async def request_scenary(session, scenary):
+    async with session.get(os.getenv(SCENARY) + scenary.lower()) as response:
+        return await response.json()
+    
+async def get_image_audio(scenary):
+    async with aiohttp.ClientSession() as session:
+        data = await request_scenary(session, scenary)
+
+        image_url = data[0][IMAGE]
+        async with session.get(image_url) as response:
+            image_data = await response.read()
+
+        audio_url = data[0][AUDIO]
+        async with session.get(audio_url) as response:
+            audio_data = await response.read()
+
+    return image_data, audio_data
+
+async def simulator(data):
```

Comment:
the method is very large, reduce the size of the method, divide it into different parts